static/js/components/GroupSources.jsx

```diff
@@ -0,0 +1,241 @@
+import React, { useEffect, Suspense } from "react";
+import PropTypes from "prop-types";
+import { useSelector, useDispatch } from "react-redux";
+
+import TableCell from "@material-ui/core/TableCell";
+import TableRow from "@material-ui/core/TableRow";
+
+import Typography from "@material-ui/core/Typography";
+import IconButton from "@material-ui/core/IconButton";
+import Grid from "@material-ui/core/Grid";
+
+import Link from "@material-ui/core/Link";
+import PictureAsPdfIcon from "@material-ui/icons/PictureAsPdf";
+
+import MUIDataTable from "mui-datatables";
+import ThumbnailList from "./ThumbnailList";
+import styles from "./RunSummary.css";
+import * as SourcesAction from "../ducks/sources";
+import { ra_to_hours, dec_to_hours } from "../units";
+
+const VegaPlot = React.lazy(() => import("./VegaPlot"));
+
+const GroupSources = ({ route }) => {
+  const dispatch = useDispatch();
+  const sources = useSelector((state) => state.sources.groupSources);
+  const groups = useSelector((state) => state.groups.user);
+
+  // Load the group sources
+  useEffect(() => {
+    dispatch(SourcesAction.fetchGroupSources({ group_id: route.id }));
+  }, [route.id, dispatch]);
+
+  if (sources === undefined || sources === null) {
+    return "Loading Sources...";
+  }
+
+  const group_id = parseInt(route.id, 10);
+  let group_name = "";
```

Comment:
```suggestion
```

Reply:
yes, indeed

Reply:
No, actually I need this in case groups is not defined.